pkg/subctl/operator/install/ensure.go

```diff
@@ -54,10 +54,13 @@ func Ensure(config *rest.Config, operatorNamespace string, operatorImage string)
 		fmt.Printf("* Updated the privileged SCC\n")
 	}
 
+	fmt.Printf("* Deploying the operator...\r")
 	if created, err := deployment.Ensure(config, operatorNamespace, operatorImage); err != nil {
 		return err
 	} else if created {
 		fmt.Printf("* Deployed the operator successfully\n")
+	} else {
+		fmt.Printf("                           \r")
```

Comment:
Ah, I was wondering what this was doing but finally see, nifty trick to "update" a status.

If anyone else is wondering: https://stackoverflow.com/a/7372938/1011749

Reply:
hahah, nice :) I still want to steal the "kind" terminal library for progress , eventually ':D